package.json

```diff
@@ -56,7 +56,11 @@
     "mongodb": "^3.6.3",
     "mongoose": "^5.11.11",
     "msw": "^0.25.0",
+    "node-fetch": "^2.6.1",
```

Comment:
node-fetch & axios accomplish the same thing. Please select one and uninstall the other

Reply:
ok checked with you. node-fetch is ok